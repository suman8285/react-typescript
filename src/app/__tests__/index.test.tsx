import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { configureAppStore } from 'store/configureStore';
import { App } from '../index';

const store = configureAppStore();

describe('App', () => {
  it('Matching the snapshot', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <App />
      </Provider>,
    );
    expect(asFragment).toMatchSnapshot();
  });
});
