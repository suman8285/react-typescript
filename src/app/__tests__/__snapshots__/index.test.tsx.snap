// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Matching the snapshot 1`] = `[Function]`;
